include/openthread-types.h

```diff
@@ -605,6 +605,16 @@ typedef struct otBorderRouterConfig
      * TRUE, if this configuration is considered Stable Network Data.  FALSE, otherwise.
      */
     bool mStable : 1;
+
+    /**
+     * The Border Agent Rloc.
+     */
+    uint16_t mRloc16;
+
+    /**
+     * The Allocated ContexId if exists.
```

Comment:
`mContextId` field would only be used on non-local network data (which would always have contextTlv for one Prefix) for below `Dhcp6AlocRlocCache`.